frontend/src/SettingsPage/SettingsPage.jsx

```diff
@@ -122,6 +154,7 @@ function SettingsPage(props) {
                         placeholder="Enter last name"
                         value={lastName}
                         onChange={(event) => setLastName(event.target.value)}
+                        onKeyPress={(event) => !/^[A-Za-z0-9<XXX> -]+$/.test(event.key) && event.preventDefault(true)}
```

Comment:
Please remove the validation, it should support special characters like é, ñ etc